### PR TITLE
Intro post with more about my background

### PR DESCRIPTION
**Pull request by jduckles**
Background post on me.

### CODE REVIEW

File: blog/2015/10/more-about-jonah-duckles.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  passionate about spreading the movement. Scientists are
+  accumulating data faster than their training has prepared them and
+  they need new approaches to raise the the level of the technical
+  conversations when thinking about complicated research challenges.
+  Software Carpentry is an incredibly powerful approach for improving
+  scientific exploration in this data-driven era. It helps scientists
+  use their existing skills and do even more. Our work is important
+  and has impact!
+</p>
+<p>
```
Comment (karinlag) [inline blog/2015/10/more-about-jonah-duckles.html]: I suggest adding something along the lines of

"For those of you who don´t know me already, I come to you from a position as XX at the University of Oklahoma. 

You then continue with "the biggest success". I think this helps with the opening, and in many ways "makes it ok" for people not to know who you are. After all, we have >300 instructors, so :)

File: blog/2015/10/more-about-jonah-duckles.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+date: 2015-10-06
+time: "12:00:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  I'm thrilled to be stepping into the role of Executive Director of
+  the Software Carpentry Foundation. I come to you as an experienced
+  instructional community member excited to share my administrative,
+  grant writing and professional backgrounds to help the Software
```
Comment (karinlag) [inline blog/2015/10/more-about-jonah-duckles.html]: background - singular

File: blog/2015/10/more-about-jonah-duckles.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  Carpentry Foundation become sustainable and reach exciting new
+  goals.
+</p>
+<!-- end excerpt -->
+<p>
+  I’ve taught Software Carpentry around the world and it has made me
+  passionate about spreading the movement. Scientists are
+  accumulating data faster than their training has prepared them and
```
Comment (karinlag) [inline blog/2015/10/more-about-jonah-duckles.html]: prepared them for

File: blog/2015/10/more-about-jonah-duckles.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+<p>
+  I work most effectively in a collaborative and entrepreneurial
+  ways, always hungering for new ideas and new ways to strategically
+  position groups of people and organizations. In just the last five
+  years, I've participated in over $30-million dollars in outside
+  funded grants with my role involving leadership, teaching and
+  mentorship in computational and data best-practices for the grants.
+  I look forward to doing similar work for Software Carpentry,
+  building grants and initiatives that can benefit The Foundation and
+  its member organizations into the future.
```
Comment (karinlag) [inline blog/2015/10/more-about-jonah-duckles.html]: Wow, really like this section above, describing where you come from!

File: blog/2015/10/more-about-jonah-duckles.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  accumulating data faster than their training has prepared them for.
+  They need new approaches to raise the the level of the technical
+  conversations when thinking about complicated research challenges.
+  Software Carpentry is an incredibly powerful approach for improving
+  scientific exploration in this data-driven era. It helps scientists
+  use their existing skills and do even more. Our work is important
+  and has impact!
+</p>
+<p>
+  For those of you the don't know me, I come to you from a position as
```
Comment (jiffyclub) [inline blog/2015/10/more-about-jonah-duckles.html]: the -> that

File: blog/2015/10/more-about-jonah-duckles.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  its member organizations into the future.
+</p>
+<p>
+  I am here to work with the entire Software Carpentry community to
+  continue the excellent work that has already been done. I invite
+  you to keep in touch with me and to give me your ideas about how we
+  can work towards helping researchers accomplish more than they
+  imagined they were capable of. I'm on twitter as <a
+  href="http://twitter.com/jduckles">@jduckles</a> and I maintain a
+  <a href="http://jduck.net">personal website</a>.
```
Comment (jiffyclub) [inline blog/2015/10/more-about-jonah-duckles.html]: Maybe add an `@software-carpentry.org` email address?

**Pull request merged**